version.json

```diff
@@ -1,3 +1,3 @@
 {
-  "version": "v0.0.7"
+  "version": "v0.1.0"
```

Comment:
I'd be grateful if you could write a commit message about why the minor upgrade, i.e. Manifest protocol version bump, breaking changes, etc. etc. 🙏